platform/rct/hal/usb_stm32/USB-CDC_device_stm32.c

```diff
@@ -1,52 +1,250 @@
-#include "USB-CDC_device.h"
```

Comment:
Please include our file header in all files you create going forward.  It is in our CODING_STANDARD.md doc.